web/src/features/progress/CircleProgressbar.tsx

```diff
@@ -100,6 +105,18 @@ const CircleProgressbar: React.FC = () => {
           <CircularProgressLabel fontFamily="Fira Mono">
             {value}%
           </CircularProgressLabel>
+          <Text
+              fontFamily="Inter"
+              isTruncated
+              fontSize={18}
+              fontWeight="light"
+              position="absolute"
+              top="115%"
+              left="50%"
+              transform="translate(-50%, -50%)"
```

Comment:
Why not just use flexbox and avoid this weird aligning? 🤷

Reply:
Wrapped it in a flex and centred it